flytekit/clients/friendly.py

```diff
@@ -1046,3 +1048,35 @@ def get_data(self, flyte_uri: str) -> _data_proxy_pb2.GetDataResponse:
 
         resp = self._dataproxy_stub.GetData(req, metadata=self._metadata)
         return resp
+
+    def get_download_deck_signed_url(
+        self,
+        node_id: str,
+        project: str,
+        domain: str,
+        name: str,
+        org: str = "",
```

Comment:
We don't need org here. The `OrgInterceptor` in the Union SDK will set the org

Reply:
This is my workflow for example.
https://serverless-1.us-east-2.s.union.ai/org/future-outlier/projects/flytesnacks/domains/development/executions/a9f6zhrwp4fzx97lzdzb/nodes

Reply:
Update: you are right, we don't need it, I tested it in a wrong way, that's why it doesn't work previously in serverless